src/sql/workbench/browser/modelComponents/declarativeTable.component.ts

```diff
@@ -298,17 +298,18 @@ export default class DeclarativeTableComponent extends ContainerBase<any, azdata
 			this._data = finalData;
 		}
 
-		const newSelectedRow = properties.selectedRow ?? -1;
-		if (newSelectedRow !== this.selectedRow && properties.enableRowSelection) {
+		const previousSelectedRow = this.selectedRow;
```

Comment:
this change has nothing to do with this issue, I am just trying to simplify the code and leave the default value logic in one place